app/controllers/counselor/groups_controller.rb

```diff
@@ -5,9 +5,10 @@ def index
   end
 
   def show
-    @current_group = Group.find(params[:id])
-    @counselors = @current_group.counselors
-    @kids = @current_group.kids
+    @group = Group.find(params[:id])
```

Comment:
Yeah, I figured that. Actually alot of pages have this problem, but for the demo it's fine nothing's going to break. 
I thought about it and I think there is way to fix it, but it's fine for now :)